src/scss/editor.scss

```diff
@@ -10,3 +10,4 @@
 @use "themes/moonlight";
 @use "themes/neulight";
 @use "themes/neudark";
+@use "themes/_gold";
```

Comment:
The `_` is not needed and add a new line at the end of file

Reply:
Oops, my bad. Fixed.